groovy/postInit/chains/StainlessSteelChain.groovy

```diff
@@ -0,0 +1,107 @@
+import static globals.Globals.*
+
+import static gregtech.api.unification.material.Materials.*;
+import gregtech.api.unification.material.MarkerMaterials;
+import static gregtech.api.unification.ore.OrePrefix.dye;
+
+
+ABS = recipemap('alloy_blast_smelter')
+VACUUM = recipemap('vacuum_chamber')
+EBF = recipemap('electric_blast_furnace')
+MIXER = recipemap('mixer')
+
+EBF.recipeBuilder()
+    .inputs(ore('dustIron') * 6)
+    .inputs(ore('dustNickel'))
+    .inputs(ore('dustTinyManganese') * 2)
+    .fluidOutputs(fluid('carburized_stainless_steel') * 1440)
+    .duration(800)
+    .EUt(Globals.voltAmps[2] * 2)
+    .buildAndRegister()
+
+EBF.recipeBuilder()
```

Comment:
Add carbon and carbon monoxide byproduct if you want direct ore to stainless charge conversion